src/Snap/Snaplet/#SqliteSimple.hs#

```diff
@@ -0,0 +1,227 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
did you accidentally add an automatic backup file in your commit, or what is this #SqliteSimple.hs# file? (you have edits to this file below in the patch)